mongo/datadog_checks/mongo/mongo.py

```diff
@@ -1165,5 +1165,5 @@ def total_seconds(td):
                 self._collect_custom_metrics_for_query(db, raw_query, custom_query_tags)
             except Exception as e:
                 metric_prefix = raw_query.get('metric_prefix')
-                self.log.warning("Errors while collecting custom metrics with prefix {}", metric_prefix)
-                self.log.warning(e)
+                self.log.warning("Errors while collecting custom metrics with prefix %s", metric_prefix)
+                self.log.exception(e)
```

Comment:
Just curious if this is to increase the error level or to have the stack trace 🤔
I'm fine with increasing the severity but then I feel like it should be the same as the previous line.
If this is to get the stack trace, then I'd rather use `self.log.warning(e, exc_info=True)`

Reply:
Good point, the log level of `log.exception()` is ERROR.

Just updated the code.